renovate.json

```diff
@@ -11,7 +11,7 @@
       "rangeStrategy": "bump"
     }
   ],
-  "ignoreDeps": ["gcp-metadata", "got", "mocha"],
+  "ignoreDeps": ["gcp-metadata", "got", "mocha", "renovate"],
```

Comment:
renovate or husky ?

Reply:
woops :)

Reply:
force pushed because it's a simple PR and the history is useless in this case